README.md

```diff
@@ -79,6 +108,8 @@ var MyComponent = onClickOutside(
 );
 ```
 
+---
```

Comment:
horizontal rules usually mean there's a missing heading, let's replace that `---` with a heading `### ensuring there's a click handler"?